src/Bicep.Core/Features/FeatureProvider.cs

```diff
@@ -20,6 +20,8 @@ public class FeatureProvider : IFeatureProvider
         private Lazy<bool> importsEnabledLazy = new(() => ReadBooleanEnvVar("BICEP_IMPORTS_ENABLED_EXPERIMENTAL", defaultValue: false), LazyThreadSafetyMode.PublicationOnly);
         public bool ImportsEnabled => importsEnabledLazy.Value;
 
+        public bool LambdasEnabled => false;
```

Comment:
Should there be an environment variable to enable lambda, or would that come in a future PR?

Reply:
I'm just going to get rid of the feature flag entirely once the ARM release has been deployed. I wanted to get this code checked in but disabled until that happens (to avoid having to deal with the conflicts while waiting for ARM to deploy).

The reason this is a feature flag and not just hard-coded as disabled is because tests need to be able to enable it.